charts/harvester-csi-driver/100.0.2+up0.1.12/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Harvester CSI Driver
+  catalog.cattle.io/kube-version: '>= 1.18'
+  catalog.cattle.io/namespace: kube-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/rancher-version: '>= 2.6.1-0 <= 2.6.99-0'
+  catalog.cattle.io/release-name: harvester-csi-driver
+  catalog.cattle.io/ui-component: harvester-csi-driver
+  catalog.cattle.io/upstream-version: 0.1.9
```

Comment:
I think this should be `0.1.12` since the version of the chart has upstream `+up0.1.12`. Also, AFAIK, this is an optional annotation and can also be removed if is not being used for anything.